package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha1-zgBCgEX7t9XrwOp7+DV4nxU2arI=",
       "requires": {
         "@babel/types": "7.0.0-beta.51",
-        "lodash": "^4.17.5"
+        "lodash": "4.17.10"
```

Comment:
Did we need to update the packages here?

Reply:
I ran npm run install a while back, and I was keeping the package-lock.json out of the commits manually, but not reverting it locally.  But I got lazy today and just did "add .", and figured it was okay to to push it up.  I will go research the changes in those lodash versions, though, to be sure there aren't any potential breaking changes.